server.js

```diff
@@ -1,7 +1,7 @@
 import express from 'express';
 import mongoose from 'mongoose';
```

Comment:
you are importing mongoose in this file but never using it...

Reply:
Also I still think ya'll should move away from EJS and handle more of this routing on the frontend w/ React Router.